app/uk/gov/hmrc/tariffclassificationfrontend/models/Application.scala

```diff
@@ -24,8 +24,24 @@ sealed trait Application {
   def asBTI(): BTIApplication = {
     this.asInstanceOf[BTIApplication]
   }
+
+  def asLiabilityOrder(): LiabilityOrder = {
+    this.asInstanceOf[LiabilityOrder]
+  }
+
+  def isBTI(): Boolean = {
+    this.isInstanceOf[BTIApplication]
+  }
+
+  def isLiabilityOrder(): Boolean = {
+    this.isInstanceOf[LiabilityOrder]
+  }
+
   def getType() : String = {
-    `type`
+    `type` match {
+      case "BTI" => "BTI"
```

Comment:
Is "BTI Application" more appropriate ?

Reply:
At the moment the requirement is just to show the type as "BTI" or "Liability Order". Potentially a question for Joe